index.html

```diff
@@ -2937,30 +2944,41 @@ <h2>Filter types for filter method
 <p>Filters may use the original values of the following bytes to
 generate the new byte value:</p>
 
-<table class="Regular" summary=
-"This table defines the variables usedin table 9.1">
+<table class="Regular numbered simple" summary=
+"This table defines the variables used in the filter types table">
+<caption>Named filter bytes</caption>
 <tr>
-<td class="Regular"><tt>x</tt> </td>
+  <th>Name</th>
+  <th>Definition</th>
+</tr>
+<tr>
+<td class="Regular">x </td>
 <td class="Regular">the byte being filtered;</td>
 </tr>
 
 <tr>
-<td class="Regular"><tt>a</tt> </td>
+<td class="Regular">a </td>
 <td class="Regular">the byte corresponding to x in the pixel immediately before the pixel containing x (or the byte immediately before x, when the bit depth is less than 8);</td>
 </tr>
 
 <tr>
-<td class="Regular"><tt>b</tt> </td>
+<td class="Regular">b </td>
 <td class="Regular">the byte corresponding to x in the previous scanline;</td>
 </tr>
 
 <tr>
-<td class="Regular"><tt>c</tt> </td>
+<td class="Regular">c </td>
 <td class="Regular">the byte corresponding to b in the pixel immediately before the pixel containing b (or the byte immediately before b, when the bit depth is less than 8).</td>
 </tr>
 </table>
 
-<p><a href="#paethpredictor-function"></a> shows the relative positions of the bytes <tt>x</tt>,
+<figure id="integer-representation-in-png">
+  <object id="figure72" data="figures/filter-bytes.svg" type="image/svg+xml" style="width: 12vw">
```

Comment:
Since this is a new figure, can we go with an id that isn't related to the section? Perhaps a descriptive name.

Reply:
Sure! It wasn't clear to me how the figure id and the object id were different.

Reply:
Oh I see, sorry that was a dumb copy-paste error to get the markup, from another figure!

Reply:
Oh sorry. I do like the figure id more now. But I actually meant the object id. Sorry about that.

What I meant is "figure72" sounds like it assumes what section it is in.

I don't think the object needs an id at all. The figure does so we can link to it with ReSpec magic. I suspect elsewhere the figure and object both have ids so we can preserve an old fragment identifier while also giving it a new-style name.

I think in this case we can just drop the object id.